.travis.yml

```diff
@@ -1,13 +1,18 @@
 language: node_js
 node_js:
-  - "4"
-  - "6"
-  - "8"
-  - "node"
+  - "8.10"
```

Comment:
Just use "8" so it uses the latest code (8.11.x as of now) vs being stuck on 8.10.x.  Add "10" too.  We usually keep "node" in there too but it does duplicate work and redo the 10 test.  Would be nice if travis de-duped itself as needed.
```
- "8"
- "10"
```

Reply:
No prob. Will fix. (Btw, I don't think this compiles on Node 10, something about ursa version)

Reply:
Two issues:
- I see you didn't add "10".  I think we really want to have that so we know we have issues to fix.
- You were a little aggressive with the 8 changes!  I was suggesting just changing the travis config to be "8" since it will by default use the latest 8.x.y, vs locking to 8.10.x and not using 8.11+.  The README and package file should still list the actual requirement of 8.10+ (or whatever it is) since we do use features only in newer releases.